packages/webdriverio/tests/scripts/isElementClickable.test.ts

```diff
@@ -199,7 +197,6 @@ describe('isElementClickable script', () => {
         global.document = { elementFromPoint: () => null } as any
 
         expect(isElementClickable(elemMock)).toBe(false)
-        expect(elemMock.scrollIntoView).toBeCalledWith({ block: 'nearest', inline: 'nearest' })
```

Comment:
Hey Christian, this was no longer needed because of the double scroll that I have changed into one. What we did was first a scroll to the top and if that wasn't enough to the center. Just like the workaround for moveTo and click, I have applied the same behaviour here and scroll once to the center 👍

Reply:
Sounds good.